core/txdb/internal/storage/snapshot.proto

```diff
@@ -9,11 +9,11 @@ message Snapshot {
   // The nodes are ordered according to a pre-order traversal.
   repeated StateTreeNode nodes = 1;
 
-  // Issuances contains the record of recent issuances for ensuring uniqueness
-  // of issuances.
-  repeated Issuance issuances = 2;
```

Comment:
I don't think so, because nonce entries, and replacement of the issuance memory with the nonce set, are an integral part of the same spec changes implemented by the rest of this PR.